pass-deposit-services/deposit-core/src/main/resources/repositories.json

```diff
@@ -31,7 +31,44 @@
         "algorithms": [
           "sha512",
           "md5"
-        ]
+        ],
+        "funder-mapping": {
```

Comment:
I'm thinking we don't want this map here in the src repositories.json since it is specific to JHU.  I think it should be removed, which means you will need to create a new test-repositories.json most likely to test your code, but I think it is cleaner to this for the community.

Reply:
Done.